src/worker/core/trade/filterItem.ts

```diff
@@ -0,0 +1,19 @@
+export default class FilterItem {
+	filterData: any; //filter items selected, ex for Pos: C, PF...
+	filterFunction: string; //function that will decide if the offer qualifies given list of players
```

Comment:
I think it would be more clear to accept the actual function here, rather than a string identifier that appears in some other file.

Reply:
So that was my original intent, but the worker seems to be pretty strict on what it accepts, i.e. it specifically precludes functions from what I can tell. Let me know if I'm missing something though!

Reply:
You're right, I didn't realize this was being called from the ui. Everything sent between goes through the "structured cloning" algorithm which does not let you easily pass functions.